package.json

```diff
@@ -14,67 +14,55 @@
     "url": "git://github.com/vigetlabs/gulp-starter.git"
   },
   "bin": {
-    "blendid": "bin/blendid.js",
-    "blendid-karma": "bin/blendid-karma.js"
+    "blendid": "bin/blendid.js"
   },
   "scripts": {
     "start": "gulp",
     "gulp": "gulp",
     "build": "gulp build",
     "deploy": "gulp deploy",
-    "test": "karma start --single-run",
-    "test:watch": "karma start",
-    "test:gulp": "mocha ./gulpfile.js/**/*.test.js"
+    "dev:test": "mocha ./gulpfile.js/**/*.test.js"
```

Comment:
this one's tests for the package itself, the others are part of the public API (sort of)